src/annoymodule.cc

```diff
@@ -408,16 +408,17 @@ py_an_on_disk_build(py_annoy *self, PyObject *args, PyObject *kwargs) {
 static PyObject *
 py_an_build(py_annoy *self, PyObject *args, PyObject *kwargs) {
   int q;
+  int n_threads = 1;
   if (!self->ptr) 
     return NULL;
-  static char const * kwlist[] = {"n_trees", NULL};
-  if (!PyArg_ParseTupleAndKeywords(args, kwargs, "i", (char**)kwlist, &q))
+  static char const * kwlist[] = {"n_trees", "n_threads", NULL};
```

Comment:
Just an aside -- what do you think about naming this parameter in the Python wrapper to `n_jobs` instead of `n_threads`? `n_jobs` seems more consistent with the Python ecosystem. Either way, I think this is a great improvement to Annoy!

Reply:
Sure, that makes sense. I can expose `n_jobs` keyword argument for Python, but keep `n_threads` everywhere else.